assets/templates/404.html

```diff
@@ -0,0 +1,28 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Page Not Found</title>
+    <link href="https://fonts.googleapis.com/css2?family=Chango&family=Roboto:wght@300&display=swap" rel="stylesheet">
```

Comment:
Is this needed for this page? Are we using icons?

Reply:
yes for the fonts that we appear on the `404` page . Otherwise the fonts are a bit weird .